.github/workflows/ci.yml

```diff
@@ -0,0 +1,115 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    # Filtering branches here prevents duplicate builds from pull_request and push
+    branches:
+      - master
+      - beta
+      - 'v*'
+      - /^greenkeeper.*$/
+
+    # Always run CI for tags
+    tags:
+      - '*'
+
+  # Early issue detection: run CI weekly on Sundays
+  schedule:
+  - cron:  '0 6 * * 0'
+
+env:
+  CI: true
+
+jobs:
+  test-mocha:
+    name: Mocha Tests
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 10.x]
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install NPM version 4
+        run: |
+          npm config set spin false
+          npm install -g npm@4
+      - name: NPM Install
+        run: npm install
```

Comment:
Should be running `yarn` not `npm install`.

See config in .travis.yml:

https://github.com/ember-fastboot/ember-cli-fastboot/blob/19f0c670913c28683ad712778f4e8a8a75deabe4/.travis.yml#L34